plugins/outputs/cloudwatchlogs/pusher.go

```diff
@@ -22,6 +22,7 @@ const (
 	reqSizeLimit     = 1024 * 1024
 	reqEventsLimit   = 10000
 	warnOldTimeStamp = 1 * 24 * time.Hour
+	logWarnInterval  = 1 * 5 * time.Minute
```

Comment:
nit: Would rename this to make it clearer that it's related to the old timestamp warning.

Reply:
Changed this to 
`
	warnOldTimeStampLogInterval = 1 * 5 * time.Minute`